react/features/base/conference/actions.js

```diff
@@ -226,6 +227,13 @@ function _addConferenceListeners(conference, dispatch) {
             id,
             email: data.value
         })));
+    conference.addCommandListener(
```

Comment:
We already have a listener for this command in this exact file. I think we should move the dispatch logic to its general command handler function.